BackofficeBundle/Resources/public/coffee/extendView/contentTypeChange.coffee

```diff
@@ -33,5 +33,5 @@ extendView['contentTypeChange'] = {
         widgetChannel.trigger 'ready', defaultValueView
         if $('#' + defaultValueId).hasClass('tinymce')
             tinymce.editors = []
-            activateTinyMce(defaultValueView, $('#' + defaultValueId))
+#            activateTinyMce(defaultValueView, $('#' + defaultValueId))
```

Comment:
Dead code here too. I stop mention it but this remark is available for all type of files :)